test/models/workers_scaling/experiment_resources_interface_test.rb

```diff
@@ -0,0 +1,129 @@
+require 'test_helper'
+require 'mocha/test_unit'
+
+class ExperimentResourcesInterfaceTest < ActiveSupport::TestCase
+
+  EXPERIMENT_ID = 'some_id'
+  USER_ID = BSON::ObjectId.new
+  # TODO change to InfrastructureId class
+  SAMPLE_INFRASTRUCTURE = ActiveSupport::HashWithIndifferentAccess.new({name: 'name', params: {foo: 'bar'}})
+  LIMIT = 10
+  SAMPLE_LIMITS = [
+      {infrastructure: SAMPLE_INFRASTRUCTURE, limit: LIMIT}
+  ]
+  DEFAULT_SCHEDULE_WORK_PARAMS = {
+      time_limit: 60
+  }
+
+  def setup
+    @sm_record_class = mock do
+      stubs(:where).returns([])
+    end
+    @experiment = mock do
+      stubs(:reload).returns(self)
+      stubs(:get_statistics).returns([100, 0, 0])
+      stubs(:experiment_size).returns(100)
+      stubs(:id).returns(EXPERIMENT_ID)
+    end
+
+    Rails.stubs(:logger).returns(stub_everything)
+    @resources_interface = WorkersScaling::ExperimentResourcesInterface.new(@experiment, USER_ID, SAMPLE_LIMITS)
+  end
+
+  test 'current_infrastructure_limit should return zero for infrastructure without limit' do
+    # given
+    resources_interface = WorkersScaling::ExperimentResourcesInterface.new(@experiment, USER_ID, {})
+    # when
```

Comment:
"then" oznacza zachowanie na które oczekujemy pisząc "should return zero". W tych testach wyrażenie then zostało jakby zignorowane, a to co ma tam być jest we "when". W przypadku kodu tych testów, właściwie to "when i then" odnosi się do jednej linijki - tej z assertami. W późniejszych testach widzę, że jest już dobrze, więc pewnie to przez pomyłkę.
Zmiana tego nie jest dla mnie istotna.